src/features/script-load.js

```diff
@@ -30,24 +30,30 @@ systemJSPrototype.createScript = function (url) {
 };
 
 // Auto imports -> script tags can be inlined directly for load phase
-var lastAutoImportUrl, lastAutoImportDeps, lastAutoImportTimeout;
+var lastAutoImportDeps, lastAutoImportTimeout;
 var autoImportCandidates = {};
 var systemRegister = systemJSPrototype.register;
+var inlineScriptCount = 0;
 systemJSPrototype.register = function (deps, declare) {
   if (hasDocument && document.readyState === 'loading' && typeof deps !== 'string') {
     var scripts = document.querySelectorAll('script[src]');
     var lastScript = scripts[scripts.length - 1];
+    var lastAutoImportUrl;
```

Comment:
Why did you move this inside of register? To avoid race conditions with multiple scripts being executed in the same tick of the event loop? I didn't think it would be possible for two inline scripts to execute before a setTimeout tick occurs.